scaper/core.py

```diff
@@ -145,7 +145,11 @@ def generate_from_jams(jams_infile, audio_outfile, fg_path=None, bg_path=None,
         ann.sandbox.scaper['bg_path'] = new_bg_path
 
     # Create scaper object
-    duration = ann.sandbox.scaper['duration']
+    if 'original_duration' in ann.sandbox.scaper:
+        duration = ann.sandbox.scaper['original_duration']
+    else:
+        duration = ann.sandbox.scaper['duration']
```

Comment:
Let's add a warning for this case (can't find original duration): 

"Couldn't find original_duration field in the scaper sandbox, using duration field instead. This can lead to incorrect behavior if generating from a jams file that has been trimmed previously"

Reply:
Got it, done - added to the test to make sure the warning happens as well.